### PR TITLE
Bookstore: connect to API

### PR DESCRIPTION
**Pull request by Ney2**
This is a PR for connecting the bookstore app with API.
1. Add dependencies for the bookstore
2. Update files to connect the bookstore with the API
3. Update the style of the bookstore

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -1,66 +1,61 @@
...
+        book.id,
+        [{
+          author: book.author,
+          title: book.title,
+          category: book.category,
+        }],
+      ],
+    };
+  },
+);
```
Comment (mateo951) [inline src/redux/books/books.js:32]: `createAsyncThunk` is used in the project for creating thunk action creators ✔

File: src/redux/books/books.js [hunk 2]
```diff
@@ -1,66 +1,61 @@
...
 
 // ACTION CREATORS
-export const addBook = (book) => ({
-  type: ADD_BOOK,
-  book,
-});
+export const addBook = createAsyncThunk(
+  ADD_BOOK,
+  async (book) => {
+    await axios.post(URL, {
```
Comment (mateo951) [inline src/redux/books/books.js:15]: All [Bookstore API](https://www.notion.so/Bookstore-API-51ea269061f849118c65c0a53e88a739) calls are done with `fetch` OR `axios` 🐐 Well done!

File: src/redux/books/books.js [hunk 3]
```diff
@@ -1,66 +1,61 @@
...
 };
 
+export const fetchBooks = createAsyncThunk(
+  FETCH_BOOKS,
+  async () => {
+    const { data } = await axios.get(URL);
+    return { books: Object.entries(data) };
+  },
+);
+
```
Comment (mateo951) [inline src/redux/books/books.js:60]: Thunk action creator for fetching books is created 🔥

**Review by mateo951 — APPROVED** [review]
### Hi @Ney2, 

Your project is complete!  There is nothing else to say other than... it's time to merge it ✔ 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/books.js (hunk 3 above)
Comment (Ney2) [reply src/redux/books/books.js:60]: @mateo951 
Thank you for your feedback

**Pull request merged**